### ISSUE TITLE
log block date/time when a new block is received.

### ISSUE THREAD

**Issue by dan-da** [Neptune-Crypto/neptune-core#73]
Presently when a new block is received from a peer, it is logged like:

```
2023-11-27T03:29:58.197404762Z  INFO ThreadId(02) neptune_core::peer_loop: Block with height 5949 is valid
```

The date/time at the left is the present time, NOT the block time.

The difference is most noticeable when performing an initial sync.   One can see the height of each received block, but there is no indicator of when it was mined, and thus you have no idea how close you are to the present/tip.

bitcoin-core *does* log the block date/time and it is super helpful when performing a 24+ hour sync.

**Comment by aszepieniec** [comment]
On a related note: the dashboard anticipates displaying a piece of information that informs the user how far he is from the present/tip: synchronization.

![image](https://github.com/Neptune-Crypto/neptune-core/assets/1583170/a62a2634-5b5a-4244-a568-c903aed59f6f)

**Comment by dan-da** [comment]
Closed by 37278c6
